website/README.md

```diff
@@ -1,531 +1,11 @@
-> import spacy
-> nlp = spacy.load("en_core_web_sm")
-> ```
-> ````
-
-```python
-### {executable="true"}
-import spacy
-nlp = spacy.load("en_core_web_sm")
-doc = nlp("This is a sentence.")
-for token in doc:
-    print(token.text, token.pos_)
-```
-
-If a code block only contains a URL to a GitHub file, the raw file contents are
-embedded automatically and syntax highlighting is applied. The link to the
-original file is shown at the top of the widget.
-
-> #### Markdown
->
-> ````markdown_
-> ```python
-> https://github.com/...
-> ```
-> ````
->
-> #### JSX
->
-> ```jsx
-> <GitHubCode url="https://github.com/..." lang="python" />
-> ```
-
-```python
-https://github.com/explosion/spaCy/tree/master/spacy/language.py
-```
-
-### Infobox {#infobox}
-
-import Infobox from 'components/infobox'
-
-> #### JSX
->
-> ```jsx
-> <Infobox title="Information">Regular infobox</Infobox>
-> <Infobox title="Important note" variant="warning">This is a warning.</Infobox>
-> <Infobox title="Be careful!" variant="danger">This is dangerous.</Infobox>
-> ```
-
-Infoboxes can be used to add notes, updates, warnings or additional information
-to a page or section. Semantically, they're implemented and interpreted as an
-`aside` element. Infoboxes can take an optional `title` argument, as well as an
-optional `variant` (either `"warning"` or `"danger"`).
-
-<Infobox title="This is an infobox">
-
-If needed, an infobox can contain regular text, `inline code`, lists and other
-blocks.
-
-</Infobox>
-
-<Infobox title="This is a warning" variant="warning">
-
-If needed, an infobox can contain regular text, `inline code`, lists and other
-blocks.
-
-</Infobox>
-
-<Infobox title="This is dangerous" variant="danger">
-
-If needed, an infobox can contain regular text, `inline code`, lists and other
-blocks.
-
-</Infobox>
-
-### Accordion {#accordion}
-
-import Accordion from 'components/accordion'
-
-> #### JSX
->
-> ```jsx
-> <Accordion title="This is an accordion">
->   Accordion content goes here.
-> </Accordion>
-> ```
-
-Accordions are collapsible sections that are mostly used for lengthy tables,
-like the tag and label annotation schemes for different languages. They all need
-to be presented – but chances are the user doesn't actually care about _all_ of
-them, especially not at the same time. So it's fairly reasonable to hide them
-begin a click. This particular implementation was inspired by the amazing
-[Inclusive Components blog](https://inclusive-components.design/collapsible-sections/).
-
-<Accordion title="This is an accordion">
-
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque enim ante,
-pretium a orci eget, varius dignissim augue. Nam eu dictum mauris, id tincidunt
-nisi. Integer commodo pellentesque tincidunt. Nam at turpis finibus tortor
-gravida sodales tincidunt sit amet est. Nullam euismod arcu in tortor auctor,
-sit amet dignissim justo congue.
-
-</Accordion>
-
-## Setup and installation {#setup}
+## Setup and installation
```

Comment:
That is an MDX syntax, that doesn't do anything in plain Markdown. GitHub auto-generates those based on the heading text and there is no way to customize them. So from a Readme-on-GitHub kind of perspective, it was a broken feature, especially, since they are just rendered as normal text which can be confusing.

[See current Readme](https://github.com/explosion/spaCy/blob/master/website/README.md)